docs/user-guide/databases.rst

```diff
@@ -0,0 +1,208 @@
+#############
+SQL Databases
+#############
+
+Felis can create a database from a schema using the command line tool or the Python API.
+This will emit DDL which can either be executed automatically or written to a file for later use.
+
+Using the Command Line Tool
+===========================
+
+The ``felis create`` command can be used to create a database from the command line.
+For more information on the command line options for the ``create`` command, see the
+`command line documentation <cli.html#felis-create>`_.
+
+Dry Run Mode
+------------
+
+The dry run mode can be used to generate SQL without actually creating the database.
+This can be useful for inspecting the SQL that will be generated before running it.
+
+There are several different ways to run in dry run mode, one being to simply omit the ``--engine-url`` option:
+
+.. code-block:: bash
+
+    felis create schema.yaml
+
+This will by default instantiate the database to an in-memory (transient) SQLite database.
+Running such a command can be useful for finding errors in the schema file before attempting to create a
+persistent database.
+
+A specific database dialect such as MySQL can be selected in dry run mode using the ``--engine-url`` option:
+
+.. code-block:: bash
+
+    felis create --engine-url mysql:// schema.yaml
+
+This will generate the SQL for a MySQL database but will not actually create it.
```

Comment:
Generate and do what? Dump it to standard output?

Reply:
Print to the console - I will clarify this.